app/views/bulk_updates/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row row-content">
+  <section>
+    <h2><%= @provider_id %> Bulk Updates</h2>
+
+    <table class="bulk-updates-list-table">
+      <thead>
+        <tr>
+          <th>Task ID</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% if @tasks.blank? %>
+          <tr>
+            <td colspan="2">No tasks found</td>
+          </tr>
+        <% else %>
+          <% @tasks.each do |task| %>
+            <tr>
+              <% next unless task['task-id'] %>
```

Comment:
No, I'm not that concerned with it. I would have taken it out but in your PR I made a comment and you said 'its a good practice to ensure you catch even unlikely scenarios'. I will leave it out, I just sometimes go back and forth figuring out where that line should be.